starburst-presto/presto.sh

```diff
@@ -35,18 +35,23 @@ function err() {
   return 1
 }
 
+# Purpose: Determine if presto is running or not
+function test_presto_with_query(){
+        # [ $(presto --server=localhost:${HTTP_PORT} --execute='select * from system.runtime.nodes;' | grep -q ${PRESTO_VERSION} && echo 0 || echo 1) -eq 0 ]
+        [ $(presto --server=localhost:${HTTP_PORT} --execute='select 1' | grep -q "1" && echo 0 || echo 1) -eq 0 ]
+}
+
 function wait_for_presto_cluster_ready() {
   # wait up to 120s for presto being able to run query
   for ((i = 0; i < 12; i++)); do
-    if presto --execute='select * from system.runtime.nodes;'; then
-      return 0
-    fi
+    # test_presto_with_query && echo "Presto is working properly." || echo "Presto is not ready, need to wait for presto cluster."
```

Comment:
Can you clean this up so there is not commented out code?

Reply:
Okay Done!.